src/containers/orgs/OrgContainer.js

```diff
@@ -178,80 +241,179 @@ class OrgContainer extends Component<Props> {
-          <Grid>
-            <GridItem>
-              <SectionTitle>Integration Account Details</SectionTitle>
-              <ItemTitle>JDBC URL</ItemTitle>
-              <div>{jdbcURL}</div>
-            </GridItem>
-            <GridItem>
-              <ItemTitle>USER</ItemTitle>
-              <div>{user}</div>
-            </GridItem>
-            <GridItem>
-              <ItemTitle>CREDENTIAL</ItemTitle>
-              <div>...</div>
-            </GridItem>
-          </Grid>
-          <Divider />
-          <Grid>
-            <GridItem>
-              <SectionTitle>Domains</SectionTitle>
-              <div>No domains</div>
-            </GridItem>
-            <GridItem>
-              <SectionTitle>Trusted Organizations</SectionTitle>
-              <div>No trusted organizations</div>
-            </GridItem>
-          </Grid>
-          <Divider />
-          <Grid>
-            <GridItem>
-              <SectionTitle>Roles</SectionTitle>
-              <SectionSubTitle>{ROLES_SUB_TITLE}</SectionSubTitle>
-              <AddNewWrapper>
-                <Input />
-                <AddButton>
-                  <FontAwesomeIcon icon={faPlus} />
-                </AddButton>
-              </AddNewWrapper>
-            </GridItem>
-            <GridItem>
-              <SectionTitle>Members</SectionTitle>
-              <SectionSubTitle>{MEMBERS_SUB_TITLE}</SectionSubTitle>
-              <AddNewWrapper>
-                <Input />
-                <AddButton>
-                  <FontAwesomeIcon icon={faPlus} />
-                </AddButton>
-              </AddNewWrapper>
-            </GridItem>
-          </Grid>
-        </CardSegment>
+          <hr />
+          {this.renderDomainsAndTrustedOrgsSection()}
+          <hr />
+          {this.renderRolesAndMembersSection()}
+        </OrgDetailsCardSegment>
       </Card>
     );
   }
 
+  renderDomainsAndTrustedOrgsSection = () => {
+
+    const { org } = this.props;
+    const { isValidDomain } = this.state;
+
+    const domains = org.get('emails', List());
+    const domainCardSegments = domains.map((emailDomain :string) => (
+      <CompactCardSegment key={emailDomain}>
+        <span>{emailDomain}</span>
+        {this.renderRemoveButton(() => {
+          this.handleOnClickRemoveDomain(emailDomain);
+        })}
+      </CompactCardSegment>
+    ));
+
+    return (
+      <TwoColumnSectionGrid>
+        <h2>Domains</h2>
+        <h2>Trusted Organizations</h2>
+        <h4>{DOMAINS_SUB_TITLE}</h4>
+        <h4>{TRUSTED_ORGS_SUB_TITLE}</h4>
+        <AddInputAddButtonRow>
+          <Input
+              invalid={!isValidDomain}
+              placeholder="Add new domain"
+              onChange={this.handleOnChangeDomain} />
+          {this.renderAddButton(this.handleOnClickAddDomain)}
+        </AddInputAddButtonRow>
+        <div>
+          <i>No trusted organizations</i>
+        </div>
+        <div>
+          {
+            domainCardSegments.count() === 0
```

Comment:
`isEmpty`?

This goes for the subsequent `.count() === 0` as well

Reply:
Did this in https://github.com/openlattice/lattice-orgs/pull/96.